server/webapp/WEB-INF/rails/webpack/views/components/tooltip/index.scss

```diff
@@ -14,40 +14,49 @@
  * limitations under the License.
  */
 @import "../../global/common";
-@import "../icons/index";
```

Comment:
@ketan -- removed explicitly importing icons styles, which was causing styles duplication.
Instead, we've changed the HTML structure of the tooltip component, so it just uses icon component.